### DISCUSSION TITLE
"Your Supabase Project ... is inactive"

### DISCUSSION THREAD

**Discussion by craignied** [supabase/supabase#30576] [Contribute to Supabase]
I got an email with this stating that my project would be shut down because it was inactive. I checked the activity logs, and there's plenty of activity. I responded to the email and filed a support ticket but crickets. I'm thinking of migrating to Google cloud because although I like Supabase's features and was giving it a go, I can't be threatened with shutting down a project for no reason.

**Comment by GaryAustin1** [comment]
What sort of activity?  Any access of the REST API's or Dashboard should count.  
Also if you are on a Pro plan it should never get paused.
